modules/virt-hot-plugging-cpu.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-edit-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-hot-plugging-cpu_{context}"]
+
+= Hot plugging CPUs on a virtual machine
+
+You can increase or decrease the number of CPU sockets allocated to a virtual machine (VM) without having to restart the VM by using the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines*.
+. Select the required VM to open the *VirtualMachine details* page.
```

Comment:
I don't have immediate access to an OCP web console, but do we really show `VirtualMachines` and not `Virtual Machines`?